runtime/port/linuxs390/j9gs.c

```diff
@@ -89,12 +90,20 @@ j9gs_initialize(struct J9PortLibrary *portLibrary, struct J9GSParameters *gsPara
 	Trc_PRT_gs_initialize_Exit(ret_code, gsParams->flags);
 
 	return IS_GS_INITIALIZED(gsParams) ? 1 : 0;
+
+#else /* !defined(J9ZTPF) */
```

Comment:
Is there a reason to have the #else section here?  Can't the ztpf code just exclude the main body and avoid duplicating the TRC and return?  Provided the gsParams is initialized before calling the function, this should be fine.

Reply:
Update made.